app/actions/websocket/teams.ts

```diff
@@ -49,3 +53,43 @@ export async function handleLeaveTeamEvent(serverUrl: string, msg: any) {
         }
     }
 }
+
+export async function handleUpdateTeamEvent(serverUrl: string, msg: WebSocketMessage) {
+    const database = DatabaseManager.serverDatabases[serverUrl];
+    if (!database) {
+        return;
+    }
+
+    const teamFromData = JSON.parse(msg.data.team);
+    const team: Team = {
+        ...teamFromData,
+    };
+
+    database.operator.handleTeam({
+        teams: [team],
+        prepareRecordsOnly: false,
+    });
+}
+
+export async function handleTeamAddedEvent(serverUrl: string, msg: WebSocketMessage) {
+    const database = DatabaseManager.serverDatabases[serverUrl];
+    if (!database) {
+        return;
+    }
+
+    const teamId = msg.data.team_id;
+    const {teams, memberships: teamMemberships} = await fetchMyTeam(serverUrl, teamId, false);
```

Comment:
@larkox @jfrerich should we start a doc for WS improvements, as in what sort of data the WS should contain and stuff like that?

Reply:
I will create a Document, share it in the Gekidou Channel can capture some of the items we discussed in the guild this morning.